tests/server-integration/SetField/SetFieldTest.hs

```diff
@@ -116,40 +98,16 @@ unit_set_field_entry_not_found = cofferTest do
         ]
       |]
 
-unit_set_field_missing_field_contents :: IO ()
-unit_set_field_missing_field_contents = cofferTest do
-  createEntry "entry"
-
-  try @HttpException
-    (
-      executeCommand
-        POST
-        ["set-field"]
-        (ReqBodyJson Null)
-        (jsonResponse @Value)
-        ( mconcat
-            [ "path" =: ("entry" :: Text)
-            , "field" =: ("field" :: Text)
-            ]
-        )
-    ) >>=
-    processStatusCodeError status400
-      [aesonQQ|
-        [
-          {
-            "error": "The entry at '/entry' does not yet have a field 'field'.\nIn order to create a new field, please include 'FIELDCONTENTS' in the body.",
-            "code": 301
-          }
-        ]
-      |]
+-- unit_set_field_missing_field_contents :: IO ()
+-- TODO: add test that expects an error in case of missing contents
```

Comment:
Seems as unsolved TODO

Reply:
Resolved now, you can check how I did it, I'm not sure it is the proper way